Dockerfile-deb-python-build

```diff
@@ -0,0 +1,52 @@
+ARG PYTHON_BUILD_NAME
+ARG BASE_IMAGE=ubuntu:22.04
+
+FROM --platform=$TARGETPLATFORM $PYTHON_BUILD_NAME-build:latest as pybuild
+
+FROM --platform=$TARGETPLATFORM $BASE_IMAGE
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TARGET_PYTHON_VERSION
+
+RUN mkdir -p /opt/yandex/ch-backup/python
+COPY --from=pybuild /opt/yandex/ch-backup/python /opt/yandex/ch-backup/python
+
+RUN set -ex \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        # Debian packaging tools
+        build-essential \
+        debhelper \
+        devscripts \
+        fakeroot \
+        # Managing keys for debian package signing
+        gpg \
+        gpg-agent \
+        # Python packaging tools
+        python3-pip \
+        python3-venv \
+        # Misc
+        curl \
+        locales \
+        # For building PyNacl library
+        libffi-dev libssl-dev libboost-all-dev libsodium-dev \
+    # Configure locales
+    && locale-gen en_US.UTF-8 \
+    && update-locale LANG=en_US.UTF-8 \
+    # Ensure that `python` refers to `python3` so that poetry works.
+    # It makes sense for ubuntu:18.04
```

Comment:
Is this comment correct? It refers to Ubuntu 18.04 but this Docerfile is based on Ubuntu 22.04 .

Reply:
Partly, but I've changed it a bit. It was supposed to use the same OS for building python as well as for building ch-backup later, so we currently use proper OS via arg